Tools/glib/api_test_runner.py

```diff
@@ -314,11 +326,24 @@ def run_tests(self):
         # Remove skipped tests now instead of when we find them, because
         # some tests might be skipped while setting up the test environment.
         self._tests = [test for test in self._tests if self._should_run_test_program(test)]
+        number_of_qt_tests = 0
+        number_of_wpe_platform_wayland_tests = 0
```

Comment:
I suggest to change this variable name to `number_of_remaining_wpe_platform_wayland_tests` because it reflects better for what is used